finality-aleph/src/metrics/mod.rs

```diff
@@ -0,0 +1,6 @@
+mod chain_status;
+mod timing;
+
+pub use chain_status::start_chain_state_metrics_job_in_current_thread;
+pub use timing::{Checkpoint, TimingBlockMetrics};
+pub(crate) const LOG_TARGET: &str = "aleph-metrics";
```

Comment:
No `pub(crate)` plx, I think this can just be without any `pub`, since it's only used inside of this module, no?